lib/wibox/widget/imagebox.lua

```diff
@@ -6,29 +6,49 @@
 -- @widgetmod wibox.widget.imagebox
 ---------------------------------------------------------------------------
 
+local lgi = require("lgi")
+local Rsvg = lgi.Rsvg
+local cairo = lgi.cairo
+
 local base = require("wibox.widget.base")
 local surface = require("gears.surface")
 local gtable = require("gears.table")
+local gdebug = require("gears.debug")
 local setmetatable = setmetatable
 local type = type
-local print = print
+local math = math
+
 local unpack = unpack or table.unpack -- luacheck: globals unpack (compatibility with Lua 5.1)
 
 local imagebox = { mt = {} }
 
+---Load rsvg handle form image file
+---@tparam string file Path to svg file.
+---@return Rsvg handle
+---@treturn number Image default width
+---@treturn number Image default height
+local function try_to_load_svg(file)
+    if not (Rsvg and file:match("%.svg$")) then return end
+
+    local handle, err = Rsvg.Handle.new_from_file(file)
+    if not handle then
+        gdebug.print_error(debug.traceback(
+            "Rsvg failed to load '" .. tostring(file) .. "': " .. tostring(err)))
+        return
```

Comment:
Should this error be turned into "the callers problem" instead of being printed immediately? I.e. `return nil, err` and then handle this in the caller?